jinja2/includes/lang_switcher.html

```diff
@@ -1,3 +1,5 @@
+{% set default_locale = settings.LANGUAGE_CODE %}
+{% set dafault_language = settings.LOCALES[default_locale].native %}
```

Comment:
please change ``dafault_language`` to ``default_language`` (``a`` to ``e``), and in the code below.